components/prize_pool/commons/prize_pool.lua

```diff
@@ -725,10 +725,17 @@ function PrizePool:_storeData()
 	for _, lpdbEntry in ipairs(lpdbData) do
 		lpdbEntry.players = mw.ext.LiquipediaDB.lpdb_create_json(lpdbEntry.players or {})
 		lpdbEntry.extradata = mw.ext.LiquipediaDB.lpdb_create_json(lpdbEntry.extradata or {})
-		local lowerCaseParticipant = mw.ustring.lower(lpdbEntry.participant)
+		local objectName = 'ranking_' .. prizePoolIndex .. '_'
```

Comment:
If not used by SMW, move it into the `storeLpdb`.
Also considering extracting it to a function.